roles/tests-integration/tests/common/mod.rs

```diff
@@ -282,3 +286,86 @@ pub async fn start_template_provider(tp_port: u16) -> TemplateProvider {
     template_provider.generate_blocks(16);
     template_provider
 }
+
+pub async fn start_sv2_translator(upstream: SocketAddr) -> SocketAddr {
+    let upstream_address = upstream.ip().to_string();
+    let upstream_port = upstream.port();
+    let upstream_authority_pubkey = Secp256k1PublicKey::try_from(
+        "9auqWEzQDVyd2oe1JVGFLMLHZtCo2FFqZwtKA5gd9xbuEu7PH72".to_string(),
+    )
+    .expect("failed");
+    let listening_address = get_available_address();
+    let listening_port = listening_address.port();
+    let hashrate = measure_hashrate(3) as f32 / 100.0;
```

Comment:
why divide by 100?

Reply:
This is done just part of the testing, I think this will change also until we figure out the best numbers here and in `measure_hashrate`